src/collection-browser.ts

```diff
@@ -1175,7 +1168,7 @@ export class CollectionBrowser
     };
 
     this.facetsLoading = true;
-    const searchResponse = await this.searchService?.search(
+    const searchResponse = await this.searchService.search(
```

Comment:
any reason why we removed optional chaining (?.) operator here?

Reply:
Good question. New line 1157 above is doing an early return if there's no search service set, so the optional chain here is no longer needed.

(Same reasoning below in `fetchPage` too).